pkg/util/validation/limits.go

```diff
@@ -33,6 +33,13 @@ type Limits struct {
 
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (l *Limits) RegisterFlags(f *flag.FlagSet) {
+	// lite uses both ingester.Config and distributor.Config.
```

Comment:
this comment is taken from 'pkg/ingester/client/client.go::RegisterFlags()' that also uses this approach.